### PR TITLE
Adds a kiss command

### PR DESCRIPTION
**Pull request by RodolfoHero**
Add a kiss command to have fun kissing your friends :b

### CODE REVIEW

**Comment by pedrofracassi** [comment]
Run `npm test`, fix the issues and commit. (You can use `npm start fix` to fix most of the issues automatically)

**Comment by pedrofracassi** [comment]
NPM FIX PLZ

File: src/commands/gifs/kiss.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    var kissnum = Math.floor(Math.random() * KissArray.length);
+    var kissImg = KissArray[kissnum];
+
+    if (msg.mentions.users.first().id != '389803234660712463') {
+        message.channel.startTyping()
+
+        const embed = new SwitchbladeEmbed(message.author)
+                                    .setColor("0x0000FF")
+                                    .setImage(kissImg)
+                                    .setDescription(`${msg.author.username} deu um beijo em ${user}`)
```
Comment (bolsomito) [inline src/commands/gifs/kiss.js]: `user` not defined afaik

File: src/commands/gifs/kiss.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  run (message) {
+            if (message.mentions.users.first().id == this.user.id) return message.channel.send('It\'s so sad looking at you alone')
+    
+
+    KissArray = new Array()
+    KissArray[0] = 'https://em.wattpad.com/be664a8e8b471f49798ad367e7e809bea6dff987/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f776174747061642d6d656469612d736572766963652f53746f7279496d6167652f5470763473634d653344323357413d3d2d3331373535353132392e3134373938323930393132363361666232353737353336373930322e676966?s=fit&w=1280&h=1280';
+    KissArray[1] = 'https://s-media-cache-ak0.pinimg.com/originals/54/31/62/5431628f9c32f0e804689c2c68db8a2a.gif';
+    KissArray[2] = 'https://s-media-cache-ak0.pinimg.com/originals/af/ec/70/afec70ee60f45a2be98ba1a76385376a.gif'
+    var kissnum = Math.floor(Math.random() * KissArray.length);
+    var kissImg = KissArray[kissnum];
```
[the remark this answers is not in the document]
Comment (bolsomito) [reply src/commands/gifs/kiss.js]: also: use `kissArray = ['', '', '']`

File: src/commands/gifs/kiss.js [hunk 3]
```diff
@@ -1,30 +1,29 @@
...
+            if (message.mentions.users.first().id == this.client.user.id) return message.channel.send('It\'s so sad looking at you alone')
     
 
-    KissArray = new Array()
-    KissArray[0] = 'https://em.wattpad.com/be664a8e8b471f49798ad367e7e809bea6dff987/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f776174747061642d6d656469612d736572766963652f53746f7279496d6167652f5470763473634d653344323357413d3d2d3331373535353132392e3134373938323930393132363361666232353737353336373930322e676966?s=fit&w=1280&h=1280';
-    KissArray[1] = 'https://s-media-cache-ak0.pinimg.com/originals/54/31/62/5431628f9c32f0e804689c2c68db8a2a.gif';
-    KissArray[2] = 'https://s-media-cache-ak0.pinimg.com/originals/af/ec/70/afec70ee60f45a2be98ba1a76385376a.gif'
-    var kissnum = Math.floor(Math.random() * KissArray.length);
-    var kissImg = KissArray[kissnum];
+    KissArray = ['https://em.wattpad.com/be664a8e8b471f49798ad367e7e809bea6dff987/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f776174747061642d6d656469612d736572766963652f53746f7279496d6167652f5470763473634d653344323357413d3d2d3331373535353132392e3134373938323930393132363361666232353737353336373930322e676966?s=fit&w=1280&h=1280',
```
Comment (pedrofracassi) [inline src/commands/gifs/kiss.js]: This can be kept out of `run()`. Also, make it a const, as it won't change.

File: src/commands/gifs/kiss.js [hunk 4]
```diff
@@ -1,30 +1,29 @@
...
 
-    if (msg.mentions.users.first().id != '389803234660712463') {
+    if (message.mentions.users.first().id !== this.client.user.id) {
         message.channel.startTyping()
 
         const embed = new SwitchbladeEmbed(message.author)
                                     .setColor("0x0000FF")
                                     .setImage(kissImg)
-                                    .setDescription(`${msg.author.username} deu um beijo em ${user}`)
+                                    .setDescription(`${message.author.username} deu um beijo em ${message.mentions.users.first().username}`)
```
Comment (pedrofracassi) [inline src/commands/gifs/kiss.js]: **It's ok to mention the users. It's an embed, they won't be pinged.** No need to use `.username`

File: src/commands/gifs/kiss.js [hunk 5]
```diff
@@ -1,30 +1,29 @@
...
-    KissArray[1] = 'https://s-media-cache-ak0.pinimg.com/originals/54/31/62/5431628f9c32f0e804689c2c68db8a2a.gif';
-    KissArray[2] = 'https://s-media-cache-ak0.pinimg.com/originals/af/ec/70/afec70ee60f45a2be98ba1a76385376a.gif'
-    var kissnum = Math.floor(Math.random() * KissArray.length);
-    var kissImg = KissArray[kissnum];
+    KissArray = ['https://em.wattpad.com/be664a8e8b471f49798ad367e7e809bea6dff987/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f776174747061642d6d656469612d736572766963652f53746f7279496d6167652f5470763473634d653344323357413d3d2d3331373535353132392e3134373938323930393132363361666232353737353336373930322e676966?s=fit&w=1280&h=1280',
+            'https://s-media-cache-ak0.pinimg.com/originals/54/31/62/5431628f9c32f0e804689c2c68db8a2a.gif',
+            'https://s-media-cache-ak0.pinimg.com/originals/af/ec/70/afec70ee60f45a2be98ba1a76385376a.gif']
+    var kissnum = Math.floor(Math.random() * KissArray.length)
+    var kissImg = KissArray[kissnum]
```
Comment (pedrofracassi) [inline src/commands/gifs/kiss.js]: Make both of these constants as well.

**Comment by pedrofracassi** [comment]
Also, **please** run `npm test` and fix all issues before commiting. We can't merge it if travis doesn't like it.

File: src/commands/gifs/kiss.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 const { Command, SwitchbladeEmbed } = require('../../index')
 
-var KissArray = ['https://em.wattpad.com/be664a8e8b471f49798ad367e7e809bea6dff987/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f776174747061642d6d656469612d736572766963652f53746f7279496d6167652f5470763473634d653344323357413d3d2d3331373535353132392e3134373938323930393132363361666232353737353336373930322e676966?s=fit&w=1280&h=1280', 'https://s-media-cache-ak0.pinimg.com/originals/54/31/62/5431628f9c32f0e804689c2c68db8a2a.gif', 'https://s-media-cache-ak0.pinimg.com/originals/af/ec/70/afec70ee60f45a2be98ba1a76385376a.gif']
+const KissArray = ['https://em.wattpad.com/be664a8e8b471f49798ad367e7e809bea6dff987/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f776174747061642d6d656469612d736572766963652f53746f7279496d6167652f5470763473634d653344323357413d3d2d3331373535353132392e3134373938323930393132363361666232353737353336373930322e676966?s=fit&w=1280&h=1280', 'https://s-media-cache-ak0.pinimg.com/originals/54/31/62/5431628f9c32f0e804689c2c68db8a2a.gif', 'https://s-media-cache-ak0.pinimg.com/originals/af/ec/70/afec70ee60f45a2be98ba1a76385376a.gif']
```
Comment (pedrofracassi) [inline src/commands/gifs/kiss.js]: Rename it to `kissarray`

Comment (RodolfoHero) [reply src/commands/gifs/kiss.js]: Done.

**Comment by pedrofracassi** [comment]
Fix standard plz

File: src/commands/gifs/kiss.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+const { Command, SwitchbladeEmbed } = require('../../index')
```
Comment (davipatury) [inline src/commands/gifs/kiss.js]: `require('../../')`

File: src/commands/gifs/kiss.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+module.exports = class Kiss extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'kiss'
+    this.aliases = ['beijo', 'beijar']
+  }
+
+  run (message) {
+    var kissnum = Math.floor(Math.random() * KissArray.length)
+    var kissImg = kissarray[kissnum]
```
Comment (davipatury) [inline src/commands/gifs/kiss.js]: Remove line 13, replace line 14 with `const kissImg = kissarray[Math.floor(Math.random() * kissarray.length)]`

File: src/commands/gifs/kiss.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+    if (message.mentions.users.first().id === this.client.user.id) return message.channel.send('It\'s so sad looking at you alone')
+    if (message.mentions.users.first().id !== this.client.user.id) {
+      message.channel.startTyping()
+      const embed = new SwitchbladeEmbed(message.author).setColor('0x0000FF').setImage(kissImg).setDescription(`${message.author} deu um beijo em ${message.mentions.users.first()}`)
+      message.channel.send(
+        embed
+      ).then(() => message.channel.stopTyping())
```
Comment (davipatury) [inline src/commands/gifs/kiss.js]:
```js
const user = message.mentions.users.first()
if(!user) return message.channel.send('usage here')
if(user.id === this.client.user.id) return message.channel.send('kiss bot')
if(user.id === message.author.id) return message.channel.send('kiss yourself')

message.channel.send(
  new SwitchbladeEmbed(message.author)
    .setColor('#0000FF')
    .setImage(kissImg)
    .setDescription(`${message.author} kissed ${user}`)
)
```

**Pull request merged**